### PR TITLE
Rename flavour_categories -> flavours

### PR DESCRIPTION
**Pull request by samvanstroud**
## Summary

This pull request introduces the following changes

* as title
*

## Conformity
- [x] [Changelog entry](https://github.com/umami-hep/puma/blob/main/changelog.md)
- [x] [Documentation](https://umami-hep.github.io/puma/)

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ### [Latest]
 
+- Rename flavour_categories -> flavours [#146](https://github.com/umami-hep/puma/pull/149)
```
Comment (afroch) [inline changelog.md:5]: The PR number and link is wrong. Should be 154

**Review by afroch — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (samvanstroud) [reply changelog.md]: thanks! resolved

**Comment by afroch** [comment]
Maybe a quick question for me. Why do we need to rename that? Just asking because I can see a problem downstream in Umami when we change this here and update the Puma version in Umami

**Comment by samvanstroud** [comment]
> Maybe a quick question for me. Why do we need to rename that? Just asking because I can see a problem downstream in Umami when we change this here and update the Puma version in Umami

@afroch yeah no real reason other than to save a few characters. I would be happy to make the changes in umami when we next bump the puma version. Otherwise if you prefer not to have this change feel free to just close the MR.

**Comment by afroch** [comment]
Yeah.. not so sure about this because I think flavours could be used already somewhere else. I'd rather close this one.

**Pull request closed without merging**